doc/releases/nautilus.rst

```diff
@@ -1,3 +1,382 @@
+v14.2.5 Nautilus
+================
+
+This is the fifth release of the Ceph Nautilus release series. Among the many
+notable changes, this release fixes a critical BlueStore bug that was introduced
+in 14.2.3. All Nautilus users are advised to upgrade to this release.
+
+Notable Changes
+---------------
+
+Critical fix:
+
+* This release fixes a `critical BlueStore bug <https://tracker.ceph.com/issues/42223>`_ 
+  introduced in 14.2.3 (and also present in 14.2.4) that can lead to data
+  corruption when a separate "WAL" device is used.
+
+New health warnings:
+
```

Comment:
* Ceph will now issue health warnings if daemons have recently crashed.  Ceph has been collecting crash reports since the initial Nautilus release, but the health alerts are new.  To view new crashes (or all crashes, if you've just upgraded),::

ceph crash ls-new

To acknowledge a particular crash (or all crashes) and silence the health warning,::

ceph crash archive <crash-id>
ceph crash archive-all

Reply:
OK, added it.